modules/speed/speed.php

```diff
@@ -0,0 +1,82 @@
+<?php
+/**
+ * Get better performance out of your WordPress site.
+ *
+ * @author Konstantin Kovshenin
+ */
+
+/**
+ * Remove slow filters from WordPress
+ *
+ * @module speed
+ */
+function jetpack_speed_init() {
+	global $wpdb;
+
+	/**
```

Comment:
Could you document the filter as done [here](https://github.com/Automattic/jetpack/blob/f6ba39802aea46da7ef2fad2bd5a09c983f77e4e/modules/related-posts/jetpack-related-posts.php#L183) for example, so our parser can grab all necessary information?

Thanks!